extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -332,28 +332,49 @@ public function processCuratedContent( $rawData ) {
 	 * @param array $data
 	 * @return array
 	 */
-	public function getCuratedContentSections( Array $data ) {
+	public function getCuratedContentSections( Array $data, $newFormat=false) {
 		$sections = [ ];
+
+		if ( $newFormat && !empty( $data[ 'sections' ] )  ) {
+			foreach ( $data[ 'sections' ] as $dataItem ) {
+				$section = [];
+				$section[ 'label' ] = $dataItem['title'];
```

Comment:
Isn't this line changing displaying logic? In the past we had `item.label || item.title` (https://github.com/Wikia/mercury/blob/ba05daf05c62a5012b7851d76f9a6d84c04ccb5b/server/app/facets/operations/prepare-curated-main-page-data.js#L46-L46). Is it intended?